### ISSUE TITLE
Tensorflow Implementation usage report

### ISSUE THREAD

**Issue by darcula1993** [PRIS-CV/Mutual-Channel-Loss#11]
I implement the tf version Mutual Channel Loss function here: https://github.com/darcula1993/Mutual-Channel-Loss.

I use it with my own 3-class fine-grained image classification task (around 90,0000 images) and it works well for now: with default hyperparameter setting, it converge much faster than before. 

The model is still training and I will keep update result here.

**Comment by dongliangchang** [comment]
Great work, I will add a link in the MC-Loss homepage to show your work.
Many Thanks.
